04-stl-basics/tasks/02-concatenateVector.cpp

```diff
@@ -3,7 +3,21 @@
 
 // Implement concatenateVector.
 // It should take 2 vectors and return one which has alternately elements from the first and the other one.
+vector<int> concatenateVector(vector<int> vec1, vector<int> vec2){
```

Comment:
```suggestion
vector<int> concatenateVector(const vector<int> & vec1, const vector<int> & vec2){
```
Jak damy const & to będzie szybciej, bo nie utworzy się dodatkowa tymczasowa kopia wektorów